Shocksupershaman - Copy/src/main/java/com/example/shocksupershaman/shocktask1/UserController.java

```diff
@@ -0,0 +1,45 @@
+package com.example.shocksupershaman.shocktask1;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.RequestParam;
+import org.springframework.web.bind.annotation.RestController;
+
+import java.util.List;
+import java.util.Optional;
+
+@RestController
+public class UserController {
+
+    @Autowired
+    private UserService userService;
+
+    @GetMapping("/getAllUsers")
+    public ResponseEntity<List<UserDTO>> getAllUsers() {
+        List<UserDTO> userDTOList = userService.getAllUsers();
+        return ResponseEntity.ok(userDTOList);
+    }
+
+    @GetMapping("/getUserById/{id}")
+    public ResponseEntity<?> getUserById(@PathVariable Long id) {
+        Optional<UserDTO> user = userService.getUserById(id);
+        return ResponseEntity.of(user);
+    }
+
+    @GetMapping("/createUser")

```

Comment:
Create should use Post endpoint and path required data inside request body.
Same about path. Endpoint should be "/" with post mapping